spec/restaurant_spec.rb

```diff
@@ -0,0 +1,96 @@
+require 'restaurant'
+
+describe Restaurant do
+	subject(:restaurant) {Restaurant.new}
+	let(:item) {"bianco"}
+	let(:another_item) {"fumo"}
+	let(:total) {4.99 + 5.80}
+
+	context "initialization" do
+		it 'creates an instance of itself' do
+			expect(restaurant).to be_an_instance_of(Restaurant)
+		end
+
+		it 'creates a pizza menu on initialization' do
+			expect(restaurant.pizza_menu).not_to be_empty
+		end
+	end
+
+	context "empty orders array" do
+		it "creates an orders empty array on initialization" do
+			expect(restaurant.orders).to be_empty
+		end
+	end
+
+	context "#add_to_cart" do
+		it "has #add_to_cart method" do
+			expect(restaurant).to respond_to(:add_to_cart).with(1).argument
+		end
+
+		it "selects an item from the pizza_menu and adds to selection array" do
+			restaurant.add_to_cart(item)
```

Comment:
Since you have put the 'add_to_cart' method under a test here, the above test which is basic, gets redundant :)